composer.lock

```diff
@@ -1543,26 +1541,20 @@
                 "mobile detector",
                 "php mobile detect"
             ],
-            "funding": [
-                {
-                    "url": "https://github.com/serbanghita",
-                    "type": "github"
-                }
-            ],
-            "time": "2021-02-19T21:22:57+00:00"
+            "time": "2022-02-17T19:24:25+00:00"
         },
         {
             "name": "monolog/monolog",
-            "version": "1.26.1",
+            "version": "1.27.1",
```

Comment:
Reminder for myself... Currently nobody uses Monolog for logging, almost everyone (including the docker images) are using either syslog or file logging

It's time to move this dependency into the addons. I think this will reduce the overall dependent libraries because Monolog is quite large